developer_setup.md

```diff
@@ -299,3 +299,32 @@ If this happens, you may be missing developer tools in your OS X. Try to install
 
 If this happens it may be a Bundler issue. Try running `bundle config specific_platform true`
 and re-try.
+
+* Can't install any gems during bundle install
```

Comment:
I think it's broad as there may be many reasons all your gems won't install. [Here's the first hit I got googling](https://stackoverflow.com/questions/2414130/cant-install-any-gems), for example.

If you don't have access and can't get the exact message, that's ok though! Thought I'd ask anyway.